plugins/modules/proxmox_backup_info.py

```diff
@@ -0,0 +1,189 @@
+  - community.general.proxmox.documentation
+  - community.general.attributes
+  - community.general.attributes.info_module
+notes:
+  - Backup specific options can be returned by this module, please look at the documentation at U(https://pve.proxmox.com/wiki/Backup_and_Restore).
+'''
+
+EXAMPLES = '''
+- name: List existing backups
+  community.general.proxmox_backup_info:
+    api_host: helldorado
+    api_user: root@pam
+    api_password: "{{ password | default(omit) }}"
+    api_token_id: "{{ token_id | default(omit) }}"
+    api_token_secret: "{{ token_secret | default(omit) }}"
+    validate_certs: "{{ validate_certs | default(omit) }}"
+    id: "{{ id | default(omit) }}"
+  register: proxmox_backups
+'''
+
+RETURN = '''
+proxmox_backups:
+  description: List of backup job.
+  returned: on success
+  type: list
+  elements: dict
+  contains:
+    mailnotification:
+      description: Notification event type
+      returned: on success
+      type: str
+    storage:
+      description: Storage used in backup job
+      returned: on success
+      type: str
+    notes-template:
+      description: Template string for generating notes for the backup(s).
+      returned: on success
+      type: str
+    mailto:
+      description: Comma-separated list of email addresses or users that should receive email notifications.
+      returned: on success
+      type: str
+    id:
+      description: Job ID.
+      returned: on success
+      type: str
+    mode:
+      description: Backup mode
+      returned: on success
+      type: str
+    next-run:
+      description: Timestamp of next run
+      returned: on success
+      type: str
+    repeat-missed:
+      description: If true, the job will be run as soon as possible if it was missed while the scheduler was not running.
+      returned: on success
+      type: bool
+    type:
+      description: Type of backup
+      returned: on success
+      type: str
+    enabled:
+      description: If true, this job is enable else it is disable the job.
+      returned: on success
+      type: bool
+    all:
+      description: If true, this job bacukup all known guest systems on this host.
+      returned: on success
+      type: bool
+    pool:
+      description: Backup mode
+      returned: on success
+      type: str
+    vmid:
+      description: The ID of the guest system backuped.
+      returned: on success
+      type: str
+'''
+
+
+class ProxmoxBackupInfoAnsible(ProxmoxAnsible):
+    def get_backup(self, id):
+        if id:
+            try:
+                backup = self.proxmox_api.cluster.backup.get(id)
+            except Exception:
+                self.module.fail_json(msg="Backup with id '%s' does not exist" % id)
+            return ProxmoxBackup(backup)
+        else:
+            backups = self.proxmox_api.cluster.backup.get()
+            backups = [ProxmoxBackup(backup) for backup in backups]
+            return backups
+
+
+class ProxmoxBackup:
+    def __init__(self, backups):
```

Comment:
Isn't the parameter information on a *single* backup? Then it should be called `backup`, not `backups`.

Reply:
Fix in commit [49d6df9541b7e86c582d5b342f3d1677f4523f5b](https://github.com/ansible-collections/community.general/pull/8301/commits/49d6df9541b7e86c582d5b342f3d1677f4523f5b)